src/SonarScanner.MSBuild.Tasks/ProjectConfig.cs

```diff
@@ -68,7 +68,7 @@ public class ProjectConfig
         /// <summary>
         /// Loads and returns project config from the specified XML file
         /// </summary>
-        internal /* for testing */ static ProjectConfig Load(string fileName) =>
+        public static ProjectConfig Load(string fileName) =>
```

Comment:
why does this need to be public?

Reply:
It wasn't accessible from tests. There's probably missing some `InternalsVisibleTo` attribute and I don't want to dig too deep in S4MSB rabbit hole.
It does no harm if it's public, just as it is in `ProjectInfo.cs`.